File: Entidades/FamPatEN.vb

Public Class FamPatEN
    Private _codfam As Integer
    Public Property CodFam() As Integer
        Get
            Return _codfam
        End Get
        Set(ByVal value As Integer)
            _codfam = value
        End Set
    End Property


    Private _codpat As Integer
    Public Property CodPat() As Integer
        Get
            Return _codpat
        End Get
        Set(ByVal value As Integer)
            _codpat = value
        End Set
    End Property

    Private _descfam As String
    Public Property DescFam() As String
        Get
            Return _descfam
        End Get
        Set(ByVal value As String)
            _descfam = value
        End Set
    End Property

    Private _descPat As String
    Public Property DescPat() As String
        Get
            Return _descPat
        End Get
        Set(ByVal value As String)
            _descPat = value
        End Set
    End Property



    Private _dvh As Integer
    Public Property DVH() As Integer
        Get
            Return _dvh
        End Get
        Set(ByVal value As Integer)
            _dvh = value
        End Set
    End Property

    Private _activo As Boolean
    Public Property Activo() As Boolean
        Get
            Return _activo
        End Get
        Set(ByVal value As Boolean)
            _activo = value
        End Set
    End Property


End Class

File: Entidades/FamPatEN.cs

namespace Entidades
{
    public class FamPatEN
    {
        private int _codfam;

        public int CodFam
        {
            get
            {
                return _codfam;
            }

            set
            {
                _codfam = value;
            }
        }

        private int _codpat;

        public int CodPat
        {
            get
            {
                return _codpat;
            }

            set
            {
                _codpat = value;
            }
        }

        private string _descfam;

        public string DescFam
        {
            get
            {
                return _descfam;
            }

            set
            {
                _descfam = value;
            }
        }

        private string _descPat;

        public string DescPat
        {
            get
            {
                return _descPat;
            }

            set
            {
                _descPat = value;
            }
        }

        private int _dvh;

        public int DVH
        {
            get
            {
                return _dvh;
            }

            set
            {
                _dvh = value;
            }
        }

        private bool _activo;

        public bool Activo
        {
            get
            {
                return _activo;
            }

            set
            {
                _activo = value;
            }
        }
    }
}